packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -145,7 +146,19 @@ export default class Java extends Target {
 const MODULE_CLASS_NAME = '$Module';
 const INTERFACE_PROXY_CLASS_NAME = 'Jsii$Proxy';
 
-const JSR305_NULLABLE = '@javax.annotation.Nullable';
+interface JavaProp {
```

Comment:
Documentation?

Reply:
Didn't understand the comment, could you clarify?

Reply:
Got it. Will add documentation to JavaProp.